### ISSUE TITLE
Change Request

### ISSUE THREAD

**Issue by kulbinderdio** [bionic-gpt/bionic-gpt#579]
Currently when setting up models the model name has to be the same as that deployed by the inference engine and this name gets displayed in the Model dropdown on the chat screen.
This can lead to confusion if the same model has been setup from multiple providers/inference instances.

A way to overcome this would be to have a unique bionic name that is used for display purposes

**Comment by tbendien** [comment]
Agreed - we need a way to provide a model unique name and ideally a description of the model so people understand which use cases to use the model for.

Users need to see context size of each model so they can set Max Tokens properly when creating Assistants.
